domain-models-runtime/src/main/java/org/folio/rest/RestVerticle.java

```diff
@@ -109,6 +109,7 @@
   public static final String        OKAPI_REQUESTID_HEADER          = "X-Okapi-Request-Id";
   public static final String        STREAM_ID                       =  "STREAMED_ID";
   public static final String        STREAM_COMPLETE                 =  "COMPLETE";
+  public static final String SYSTEM_USER_ID = "00000000-0000-0000-0000-000000000000";
```

Comment:
The system user should not have the nil UUID.
https://en.wikipedia.org/wiki/Universally_unique_identifier#Nil_UUID

Reply:
So just create some UUID for the system instead?

Reply:
@adamdickmeiss 
> The system user should not have the nil UUID
What kind of problems we can find using UUID nil value? 
But we will be sure that the system will not randomly generate a user with the same UUID as the system user. Although the chance is extremely small for this case

Reply:
If we are intending to use a known nil value, or make up a well known but meaningless value, what does that provide over omitting the property entirely?

The UI attempts to resolve these IDs as references to users, so both of these will fail.